src/libs/invoke/invoke.c

```diff
@@ -85,16 +85,24 @@ ElektraInvokeHandle * elektraInvokeOpen (const char * elektraPluginName, KeySet
 		config = ksDup (config);
 	}
 
+	if (!errorKey)
+	{
+		errorKey = keyNew (0, KEY_END);
+	}
+	else
+	{
+		keyIncRef (errorKey);
```

Comment:
In every case keyDel (errorKey) is called. If the user has passed a key, the reference is increased beforehand so the user's passed key won't get deleted at this point. If the user passed 0, a key was created inside the invoke library and it will be deleted.